config/routes.rb

```diff
@@ -1,4 +1,6 @@
 Webplatform::Application.routes.draw do
+
+  resource :user_sessions
```

Comment:
so, I'm not sure if you were using 'rake routes' command, but try to run it in shell in your project directory. You should see all the links/paths that are available in our project. Look for 'user_sessions#new' and 'user_sessions#destroy'. They both appears 2 times, it's because 'resource' generated its own paths for this actions, but we already added our own paths (\login and 'logout) to this actions. As we don't need to generate all of those routes and we don't need duplication, lets add some additional information to 'resource'. So it will be:

 resources :user_sessions, only: [:create]

try now to run 'rake routes'. Maybe it is just a cosmetic change, but its not so good to create all this routes when we don't need them :)